Core/KeyMap.cpp

```diff
@@ -395,6 +395,11 @@ const KeyMap_IntStrPair psp_button_names[] = {
 	{VIRTKEY_ANALOG_ROTATE_CW, "Rotate Analog (CW)"},
 	{VIRTKEY_ANALOG_ROTATE_CCW, "Rotate Analog (CCW)"},
 
+	{VIRTKEY_SCREEN_ROTATION_VERTICAL, "Portrait"},
+	{VIRTKEY_SCREEN_ROTATION_VERTICAL180, "Portrait reversed"},
+	{VIRTKEY_SCREEN_ROTATION_HORIZONTAL, "Landscape"},
+	{VIRTKEY_SCREEN_ROTATION_HORIZONTAL180, "Landscape reversed"},
```

Comment:
Will these be descriptive enough in the mapping UI? Although I guess "Screen Orientation Portrait Reverse" would be too long...

Reply:
Maybe just "Display Portrait" or "Orient Portait" etc.?  Super long descriptions get awkward in that UI...

-[Unknown]

Reply:
Went for "Display ..." it wrap to new line in the reversed one, but it already happen for other language with other keys AFAIR.